recipes/libnice/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+import os
+from conan import ConanFile
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.files import copy, get, rmdir, rename, chdir, replace_in_file
+from conan.tools.layout import basic_layout
+from conan.tools.gnu import PkgConfigDeps
+from conan.errors import ConanInvalidConfiguration
+
+
+class LibniceConan(ConanFile):
+    name = "libnice"
+    version = "0.1.19"
```

Comment:
```suggestion
```
Please, do not set a version in the recipe, so it can be re-used for any version.